src/main/java/adris/altoclef/util/KillAura.java

```diff
@@ -19,6 +22,7 @@
     private final TimerGame _hitDelay = new TimerGame(0.2);
     private double _forceFieldRange = Double.POSITIVE_INFINITY;
     private Entity _forceHit = null;
+    private final MinecraftClient _mc = MinecraftClient.getInstance();
```

Comment:
Maybe this is a good convention, but I usually like to limit instance variables if I can.

I'd move this to a local variable either at the top of `tickStart` or at the top of `case DELAY:` near lines 27 or 62.

Reply:
moved the variable to the top of `tickStart`